x-pack/plugins/apm/public/context/LoadingIndicatorContext.tsx

```diff
@@ -26,9 +26,9 @@ function reducer(statuses: State, action: Action) {
   // Return an object with only the ids with `true` as their value, so that ids
   // that previously had `false` are removed and do not remain hanging around in
   // the object.
-  return pick(
+  return pickBy(
     { ...statuses, [action.id.toString()]: action.isLoading },
-    Boolean
+    isBoolean
```

Comment:
@smith can you verify whether this is correct?

Reply:
... my understanding is that `Boolean` will cast to a boolean value whereas `isBoolean` will check whether the given value is a boolean. So it looks like this change modified the existing behaviour.
@smith do we have any tests unit for this?

Reply:
my bad, I'll recover the original behaviour here!

Reply:
alright, done!